dc-model-jackson/src/test/resources/serializedTestObjects/identifiable/entity/DigitalObject.json

```diff
@@ -1,6 +1,7 @@
 {
   "identifiableObjectType" : "DIGITAL_OBJECT",
   "identifiers" : [ {
+    "objectType" : "identifier",
```

Comment:
Vielleicht auch besser in Großbuchstaben?

Reply:
done